plugins/opds.koplugin/opdsbrowser.lua

```diff
@@ -695,7 +695,40 @@ function OPDSBrowser:downloadFile(item, filename, remote_url)
     end
 end
 
-function OPDSBrowser:streamPages(item, remote_url, count)
+-- This function shows a dialog with input fields
+-- for entering information for Page streaming from.
+-- an input page number
+function OPDSBrowser:jumpToPage(viewer)
+    local input_dialog
+    input_dialog = InputDialog:new{
+        title = _("Enter Page Number"),
+        input = "",
```

Comment:
The line can be removed

Reply:
Leaving a note for myself, this is done.